figo.gemspec

```diff
@@ -2,9 +2,9 @@
 
 Gem::Specification.new do |s|
   s.name          = "figo"
-  s.version       = "2.0.13"
-  s.authors       = ["figo GmbH", "Netguru"]
-  s.email         = ['devs@figo.io', "lukasz.ozimek@netguru.com", "robert.kusmirek@netguru.com"]
+  s.version       = '2.0.14'
```

Comment:
I think you'll want this to be `2.0.15`, since the `2.0.14` tag already exists and it seems like this file was just forgotten last time.